src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlQueueClient.cs

```diff
@@ -0,0 +1,287 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Model;
+using Microsoft.Health.JobManagement;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Storage;
+using JobStatus = Microsoft.Health.JobManagement.JobStatus;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage
+{
+    public class SqlQueueClient : IQueueClient
```

Comment:
This file has a lot of overlap with the SqlServerFhirOperationDataStore. When orchestrator and process jobs are added to export I think these two files will merge.

Reply:
Thanks, agree that they should merge to one.